dftimewolf/cli/recipes/artifact_grep.py

```diff
@@ -0,0 +1,52 @@
+"""Collect artifacts from hosts using GRR.
+
+- Collect a predefined list of artifacts from hosts using GRR
+- Process them locally with grep to extract keywords
+"""
+from __future__ import unicode_literals
+
+_short_description = ('Fetches requested artifacts from a list of GRR hosts '
+                      'and processes them with grep.')
```

Comment:
I'd try to be a bit more explicit here, such as "runs grep with a list of keywords on them"

Reply:
Done